packages/access-client/src/utils/did-mailto.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} email
+ * @returns {`did:mailto:${string}:${string}`}
+ */
+export function createDidMailtoFromEmail(email) {
+  const emailParts = email.split('@')
+  if (emailParts.length !== 2) {
+    throw new Error(`unexpected email ${email}`)
+  }
+  const [local, domain] = emailParts
+  const did = /** @type {const} */ (
+    `did:mailto:${encodeURIComponent(domain)}:${encodeURIComponent(local)}`
```

Comment:
Nit: We should check that neither part is empty

Reply:
If the did-mailto syntax banned empty segments, I would add that, but I think it is relatively undefined.
I would be in favor of `email` opt being an already-validated `Email` obj, and that may make sense as a check when parsing a string into an email.